types/src/validator_verifier.rs

```diff
@@ -576,12 +626,12 @@ mod tests {
         partial_sig.add_signature(unknown_validator_signer.author(), unknown_signature);
 
         let (multi_sig, _) = unknown_validator
-            .aggregate_multi_signature(&partial_sig)
+            .aggregate_signatures(&partial_sig)
             .unwrap();
 
         assert_eq!(
             validator.verify_multi_signatures(&dummy_struct, &multi_sig),
-            Err(VerifyError::InvalidSignature)
+            Err(VerifyError::InvalidMultiSignature)
```

Comment:
InvalidAggregateSignature

Reply:
This looks fine to me - it should throw invalid multi sig error right?

Reply:
ah i didn't realize there's both invalid multi sig and invalid aggr sig